R/expect_s3_class_linter.R

```diff
@@ -5,6 +5,19 @@
 #'   and [testthat::expect_true()] can also be used for such tests,
 #'   but it is better to use the tailored function instead.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "expect_equal(class(x), 'data.frame')",
```

Comment:
`expect_equal(class(x), "numeric")`

vs.

`expect_type(x, "double")`

being the replacement would also be good to highlight here.

Reply:
Good suggestion. I have included it as an additional example.